packages/gatsby-plugin-feed/README.md

```diff
@@ -86,6 +70,10 @@ plugins: [
 ]
 ```
 
-To see what option keys are valid, see [the itemOptions section](https://www.npmjs.com/package/rss#itemoptions) of the RSS module.
+Each feed must include `output`, `query`, and `title`. Additionally, it is strongly recommended to pass a custom `serialize` function, otherwise an internal serialize function will be used which may not exactly match your particular use case.
```

Comment:
@marcysutton is this documented OK? I think this could maybe be improved, so suggestions more than welcome!

Also probably a good idea to link to your new doc in this README--when it's created/merged!

Reply:
This is great and way more helpful than before!